docs/items/index.md

```diff
@@ -1,72 +1,127 @@
-Items
-=====
+# Items
 
-Along with blocks, items are a key component of most mods. While blocks make up the level around you, items exist within inventories.
+Along with blocks, items are a key component of Minecraft. While blocks make up the world around you, items exist within inventories.
 
-Creating an Item
-----------------
+## What Even Is an Item?
 
-### Basic Items
+Before we get further into creating items, it is important to understand what an item actually is, and what distinguishes it from, say, a [block][block]. Let's illustrate this using an example:
 
-Basic items that need no special functionality (think sticks or sugar) do not need custom classes. You can create an item by instantiating the `Item` class with an `Item$Properties` object. This `Item$Properties` object can be made via the constructor and customized by calling its methods. For instance:
+- In the world, you encounter a dirt block and want to mine it. This is a **block**, because it is placed in the world. (Actually, it is not a block, but a blockstate. See the [Blockstates article][blockstates] for more detailed information.)
+- Once you have mined the block, it is removed (= replaced with an air block) and the dirt drops. The dropped dirt is an item **entity**. This means that like other entities (pigs, zombies, arrows, etc.), it can inherently be moved by things like water pushing on it, or burned by fire and lava.
```

Comment:
I guess this is technically true, but it might be better practice to leave a link to loot tables when we say the block 'drops'.